src/api/api.js

```diff
@@ -1,10 +1,17 @@
 import apiUrl from './apiConfig';
 import axios from 'axios';
 
-export const getEpisodes = async (searchTerm) => {
+export const getData = async (titleID) => {
   return await axios({
     method: 'GET',
-    url: apiUrl + `/episode-data/${searchTerm}`,
+    url: apiUrl + `/data/${titleID}`,
```

Comment:
these routes in the backend should probably be renamed to `get-data`, `get-episode-data`, `get-meta-data` and so on